index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <script src="redux.min.js"></script>
+  <title>Changing clothes</title>
+</head>
+<body>
+  <ul class="list"></ul>
+  <script src="main.js"></script>
+</body>
+</html>
```

Comment:
the document must end with one empty line
```suggestion
</html>

```